createcard.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+
+export default class CreateCard extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      topic: '',
+      sideA: '',
+      sideB: '',
+      confirmationMessage: null
+    }
+    this.clearConfirmationMessage = this.clearConfirmationMessage.bind(this)
+  }
+
+  handleChange(e, id) {
```

Comment:
The `event.target` is an HTML element, so you can just get its `id` directly instead of taking an extra parameter.